src/onboarding/registration/constants.ts

```diff
@@ -0,0 +1,1005 @@
+  'Ominous',
+  'Omnipresent',
+  'Ongoing',
+  'Operatic',
+  'Operational',
+  'Optimal',
+  'Orbital',
+  'Ornamental',
+  'Outdated',
+  'Overdue',
+  'Overhanging',
+  'Overweight',
+  'Overwhelming',
+  'Painless',
+  'Pallid',
+  'Papal',
+  'Parasitic',
+  'Passing',
+  'Passive',
+  'Pastel',
+  'Petite',
+  'Photographic',
+  'Plentiful',
+  'Plush',
+  'Poorly',
+  'Portentous',
+  'Positive',
+  'Possessive',
+  'Potted',
+  'Practiced',
+  'Preposterous',
+  'Presumed',
+  'Prime',
+  'Producing',
+  'Productive',
+  'Profitable',
+  'Projective',
+  'Prolonged',
+  'Promising',
+  'Prompt',
+  'Pronounced',
+  'Prophetic',
+  'Prosperous',
+  'Protruding',
+  'Psychic',
+  'Puny',
+  'Pure',
+  'Puzzling',
+  'Quivering',
+  'Raised',
+  'Rampant',
+  'Rapt',
+  'Ravaged',
+  'Reassuring',
+  'Receptive',
+  'Recorded',
+  'Reflex',
+  'Reformist',
+  'Refreshing',
+  'Regal',
+  'Regulated',
+  'Repetitious',
+  'Residual',
+  'Resounding',
+  'Responsible',
+  'Restoring',
+  'Reticent',
+  'Rightful',
+  'Rolled',
+  'Rolling',
+  'Ruffled',
+  'Rustic',
+  'Salty',
+  'Salutary',
+  'Sardonic',
+  'Scanty',
+  'Scarce',
+  'Scholastic',
+  'Scientific',
+  'Scriptural',
+  'Sealed',
+  'Secure',
+  'Sedentary',
+  'Sensitive',
+  'Serial',
+  'Sharpened',
+  'Shocked',
+  'Sick',
+  'Singular',
+  'Skeptical',
+  'Skilled',
+  'Slanted',
+  'Sleek',
+  'Sleepy',
+  'Slender',
+  'Slight',
+  'Sluggish',
+  'Small',
+  'Smart',
+  'Smiling',
+  'Smoky',
+  'Smooth',
+  'Snap',
+  'Social',
+  'Socialist',
+  'Sophisticated',
+  'Sore',
+  'Sorrowful',
+  'Southwestern',
+  'Specific',
+  'Speckled',
+  'Speeding',
+  'Spent',
+  'Spoken',
+  'Spongy',
+  'Spotted',
+  'Spurious',
+  'Stalwart',
+  'Static',
+  'Stopped',
+  'Strangled',
+  'Stray',
+  'Stupid',
+  'Submerged',
+  'Subordinate',
+  'Subsidiary',
+  'Succulent',
+  'Suffering',
+  'Suggestive',
+  'Supernatural',
+  'Sure',
+  'Surly',
+  'Sustained',
+  'Sweet',
+  'Swelling',
+  'Sympathetic',
+  'Syntactic',
+  'Tacit',
+  'Tailored',
+  'Theological',
+  'Thermal',
+  'Threadbare',
+  'Tight',
+  'Touching',
+  'Trampled',
+  'Transitional',
+  'Traumatic',
+  'Trusting',
+  'Trustworthy',
+  'Trying',
+  'Turning',
+  'Typical',
+  'Tyrannical',
+  'Unabashed',
+  'Unable',
+  'Unanticipated',
+  'Unaware',
+  'Unbalanced',
+  'Uncommitted',
+  'Uncommon',
+  'Uncontrollable',
+  'Undecided',
+  'Undercover',
+  'Underlying',
+  'Understandable',
+  'Undesirable',
+  'Undulating',
+  'Unencumbered',
+  'Unfamiliar',
+  'Unfocused',
+  'Uninhibited',
+  'Unjustified',
+  'Unpaid',
+  'Unpopular',
+  'Unquestioned',
+  'Unrealistic',
+  'Unruffled',
+  'Unsafe',
+  'Unsatisfactory',
+  'Unsettled',
+  'Unsold',
+  'Unsuccessful',
+  'Untenable',
+  'Uprooted',
+  'Upstate',
+  'Urbane',
+  'Vacant',
+  'Vengeful',
+  'Very',
+  'Vicious',
+  'Vile',
+  'Virtuous',
+  'Wayward',
+  'Weird',
+  'Windy',
+  'Winsome',
+  'Wiry',
+  'Wispy',
+  'Withered',
```

Comment:
"withered", along with "vile", "vicious", "flabby", and "fake", should not be left in here. 

can you please go through and delete the negative sounding words? holding the command key and pressing backspace deletes the line. So this can all be done pretty quickly with your keyboard (no annoying triple double click + delete for each line)

Reply:
Deleted many of them and replaced them with more positive words. Let me know if you think more should be changed out.